list_ec2.py

```diff
@@ -0,0 +1,7 @@
+import boto3
```

Comment:
use ```click``` library, which will accept instance state as an command line argument, if the argument is ```Running``` than just lisr running instances, otherwise all.

Reply:
@RaghuvirDav9 I think this is pending, right?